prisma/seedDev.mjs

```diff
@@ -79,12 +79,18 @@ function randomSpecialist({ districts, specializations, therapies }) {
   }
   const specialistTherapies = uniqueObjectsWithId(therapies);
   const phoneRegexp = '+380[0-9]{9}';
-
   const socialMediaLinks = generateSocialMediaLinks();
+  const specializationsIds = uniqueObjectsWithId(specializations);
+  const specializationMethodsIds = uniqueObjectsWithId(
+    specializationMethods.filter(method => specializationsIds.find(s => s.id === method.specializationId)),
```

Comment:
Would it make sense to use Array#includes instead of Array#find?

Reply:
to use Array#includes, there will be a need to add additional Array#map to the piping to turn const specializationsIds = [{id: ...}, {id: ...}] into array of string id's. Instead of this, I replaced Array#find with Array#some. 

```js
  const specializationMethodsIds = uniqueObjectsWithId(
    specializationMethods.filter(({ specializationId }) =>
      specializationsIds.some(({ id }) => id === specializationId),
    ),
  );
```